projet/projects/views.py

```diff
@@ -69,35 +97,12 @@ class AcceptContributor(generics.RetrieveUpdateDestroyAPIView):
     def perform_update(self, serializer):
         serializer.save(accepted_by=self.request.user, date_accepted=timezone.now())
 
-    def patch(self, request, project_id=None, contributor_id=None):
```

Comment:
The two views 
```
Accept contributor
``` 
and 
``` 
contribute
``` 
were removed since the logic of the whole contribution system changed.